gdf.js

```diff
@@ -682,6 +689,7 @@ export class GDF {
     let port = 3000
 
     if (this.gatsby) port = 8080
+    if (this.vite) port = 80
```

Comment:
In conjuction with the comment above, change this to check `this.runtime` for "vite".  This will prevent Remix/vite apps from being on port 80.